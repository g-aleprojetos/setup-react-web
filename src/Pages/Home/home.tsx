import React from 'react';
import manutencao from 'assets/image/manutencao.svg';

import * as S from './home.styles';

export const Home = () => {
  return (
    <S.Container data-testid={'test_home'}>
      <S.Imagem data-testid={'test_image'} src={manutencao} />
    </S.Container>
  );
};
