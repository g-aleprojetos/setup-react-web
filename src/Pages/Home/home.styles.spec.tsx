import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import 'jest-styled-components';
import manutencao from 'assets/image/manutencao.svg';
import * as S from './home.styles';

describe('Home.styles', () => {
  test('Container DEVE ser igual ao snapshot', () => {
    const container = renderer.create(<S.Container />).toJSON();
    expect(container).toMatchSnapshot();
  });
  test('Image DEVE ser igual ao snapshot', () => {
    const imagem = renderer.create(<S.Imagem src={manutencao} />).toJSON();
    expect(imagem).toMatchSnapshot();
  });
});
