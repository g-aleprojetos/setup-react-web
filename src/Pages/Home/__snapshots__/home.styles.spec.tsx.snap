// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.styles Container DEVE ser igual ao snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
  height: 100%;
}

<div
  className="c0"
/>
`;

exports[`Home.styles Image DEVE ser igual ao snapshot 1`] = `null`;
